pysatMissions/instruments/_core.py

```diff
@@ -22,3 +23,22 @@ def _download(date_array, tag, sat_id, data_path=None):
     """ Data is simulated so no download routine is possible. Simple pass
     function"""
     pass
+
+
+def _get_times(fnames, sat_id):
+    """Construct list of times for simulated instruments"""
+
+    # grab date from filename
+    parts = os.path.split(fnames[0])[-1].split('-')
```

Comment:
Same concern about fnames[0] as listed in another pull.

Reply:
This one we have more control over.  I'm going to simplify this here as a single fname, but ultimately we will import from pysat once that feature is versioned over there.